library/choose_disk.py

```diff
@@ -412,10 +421,23 @@ def select_only_free_devices(physical_disks):
                     # Saving the ceph type (journal and/or data)
                     ceph_disk += disk_type
 
+            # If we reach this point, it means that no partition had ceph on it
+            # Having partitions is a show stopper
             if not ceph_disk:
                 logger.info('Ignoring %10s : device has existing partitions', physical_disk)
                 continue
 
+        # If we reach here and ceph_disk is not populated, it means it could be an LVM
+        # We didn't checked LVM before as some ceph disks could be under LVM
+        # disk_label is supposed to catch this case, so it only remain LVM made by the user
+        if not ceph_disk:
+            # Does the disk belongs to a LVM ?
+            return_code = os.system("pvdisplay -c /dev/{}".format(physical_disk))
```

Comment:
please avoid using `os.system`. This command will send all its output to stdout without having a way to silence it. This line only care about the return code, and subprocess can certainly handle that

Reply:
roger that. Fixed.
Thx for the insightful review.